GitOut/Themes/MaterialDesignIcons/MaterialDesignIcons.xaml

```diff
@@ -48,6 +48,7 @@
     <Geometry x:Key="ArchiveArrowUp">
         M4 21H20V8H4M14 15V18H10V15H7L12 10L17 15M3 3H21V7H3
     </Geometry>
+    <Geometry x:Key="ArrowCollapseRight">M12.08,4.08L20,12L12.08,19.92L10.67,18.5L16.17,13H2V11H16.17L10.67,5.5L12.08,4.08M20,12V22H22V2H20V12Z</Geometry>
```

Comment:
I'm contemplating if a cross icons would be more intuitive
![image](https://user-images.githubusercontent.com/38684932/115791001-f63dd980-a3c7-11eb-9b59-ae3282491bb9.png)
thoughts?

Reply:
I like the current one better, but I agree that a close button would also be suitable.
For me the "Arrow collapse right" is clearer as it is in a toolbar. If we had a button in the _details pane_ a cross would mean "close this", but in this case it's not the toolbar we're closing per se.